tests/e2e/mvp_test.go

```diff
@@ -113,8 +113,22 @@ func TestMVP(t *testing.T) {
 
 	assert.Equal(t, 70_000_000, providerCli.QueryVaultFreeBalance())
 
-	// Cross Stake - A user pulls out additional liens on the same collateral "cross staking" it on different chains.
+	// @alpe these lines pass, but when I run them, the following providerCli.MustExecVault(execMsg) fails with
```

Comment:
Good call. I will add fix to wasmd and make this work here.

Reply:
I would be happy for merge and release of this (so we have 0.3.0 mesh-security-sdk that can be tested).

And then the wasmd stuff afterwards in a patch release. Test works and shows it is enforced, but it breaks others. This is more about maintainability than correctness of current code, so can add to a patch